internal/controller/vrg_volrep.go

```diff
@@ -1424,7 +1423,7 @@ func (v *VRGInstance) validateVRStatus(volRep *volrep.VolumeReplication, state r
 	}
 
 	// it should be completed
-	conditionMet, msg := isVRConditionMet(volRep, volrepController.ConditionCompleted, metav1.ConditionTrue)
+	conditionMet, msg := isVRConditionMet(volRep, volrep.ConditionCompleted, metav1.ConditionTrue)
```

Comment:
This constant is not define anywhere in this commit - did you forget to add the file?

Reply:
I thought that csi-addons 0.9.0 did not expose the volrep constants and we need to consume a specific commit?

Reply:
Yes, volrep constants were exposed in specific commit. I am using this commit in go.mod